.github/workflows/docker-build-push.yaml

```
@@ -1,17 +1,43 @@
+  changes:
+    runs-on: arc-arm64
+    outputs:
+      proompteng: ${{ steps.filter.outputs.proompteng }}
+      findbobastore: ${{ steps.filter.outputs.findbobastore }}
+      kitty-krew: ${{ steps.filter.outputs.kitty_krew }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Check for file changes
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.4)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)